sdk/src/main/java/com/qonversion/android/sdk/internal/mappers/UserMapper.kt

```diff
@@ -0,0 +1,41 @@
+package com.qonversion.android.sdk.internal.mappers
+
+import com.qonversion.android.sdk.internal.extensions.mapping.getString
+import com.qonversion.android.sdk.internal.extensions.mapping.getDate
+import com.qonversion.android.sdk.internal.extensions.mapping.getMap
+import com.qonversion.android.sdk.public.Entitlement
+import com.qonversion.android.sdk.public.User
+import com.qonversion.android.sdk.public.UserPurchase
+
+class UserMapper internal constructor(
```

Comment:
what is the need of internal constructor? We say that objects of this class can be created only in this package, but used anywhere. Is it the desired behavior?